If there seems to be more than one possible primary device, just
pick the first device and carry on (LP 459512)

signed-off-by: Bryce Harrington <bryce@canonical.com>

diff --git a/hw/xfree86/common/xf86pciBus.c b/hw/xfree86/common/xf86pciBus.c
index ac018e7..c00391c 100644
--- a/hw/xfree86/common/xf86pciBus.c
+++ b/hw/xfree86/common/xf86pciBus.c
@@ -141,8 +141,8 @@ xf86PciProbe(void)
 		    primaryBus.id.pci = info;
 		} else {
 		    xf86Msg(X_NOTICE,
-			    "More than one possible primary device found\n");
-		    primaryBus.type ^= (BusType)(-1);
+			    "More than one possible primary device found.  Using first one seen.\n");
+                    break;
 		}
 	    }
 	}
